src/main/java/com/codecool/virtualstylist/user/Session.java

```diff
@@ -10,7 +10,7 @@
     private Integer id;
     private String  sessionId;
     private Date date;
-    @ManyToOne
+    @ManyToOne(fetch = FetchType.LAZY,cascade = CascadeType.ALL)
```

Comment:
Please remove cascade type all as this means that by removing session, user is deleted as well; Please remove fetch type Lazy as it is default for Hibernate